armada/tiller.py

```diff
@@ -99,6 +105,13 @@ def update_release(self, chart, dry_run, name, disable_hooks=False,
         else:
             values = Config(raw=values)
 
+        print name
+        for job in jobs:
+            # LOG.info("Deleting [%s] : %s", "job", items.name)
+            print name, job
+            body = client.V1DeleteOptions()
+            self.k8s.api_client.delete_namespaced_job(name, job, body)
```

Comment:
TODO:
- [ ] update the yaml for extensive fuctionality
- [ ] flags to disable pre/post upgrade actions
- [ ] if update has no diff then dont realease